interface/src/app/mlmodel/resource.js

```diff
@@ -0,0 +1,9 @@
+angular.module('adage.mlmodel.resource', ['ngResource'])
+
+.factory('MLModel', ['$resource', function($resource) {
```

Comment:
Sounds good to me. As @mhuyck pointed out earlier, another option is not to create this service at all. Just create a resource inside your component code, since it is so simple and it is not being used anywhere else in the code.